Anzee.xcodeproj/project.pbxproj

```diff
@@ -19,6 +19,7 @@
 		76CE62B3225466D0007376BB /* APIRequest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 76CE62B2225466D0007376BB /* APIRequest.swift */; };
 		76CE62B522546706007376BB /* AnzeeAPI.swift in Sources */ = {isa = PBXBuildFile; fileRef = 76CE62B422546706007376BB /* AnzeeAPI.swift */; };
 		76CE62B822546827007376BB /* CampaignsRequest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 76CE62B722546827007376BB /* CampaignsRequest.swift */; };
+		B95896C42278D93500F7509B /* Result.swift in Sources */ = {isa = PBXBuildFile; fileRef = B95896C32278D93500F7509B /* Result.swift */; };
```

Comment:
I could have sworn I saw this previously... I don't see this actual file in the repo.

Reply:
My mistake. Added to git.